test/functional/ui/float_spec.lua

```diff
@@ -5381,15 +5381,15 @@ describe('floatwin', function()
             }, "NW", 1, 0, 11, true }
         }}
       else
-        screen:expect([[
-          # TODO: 测 {2: x x  x}{1:息}{2: xx} 确性            |
+        screen:expect{grid=[[
+          # TODO: 测试{2: x x  x}{1:息}{2: xx}确性            |
           # FIXME: 测{1:试}{2:x x  x}{1:息}{2: x}准确^性           |
           {3:~          }{4:            }{3:                 }|
           {3:~                                       }|
           {3:~                                       }|
           {3:~                                       }|
                                                   |
-        ]])
+        ]], timeout=10000}
```

Comment:
This test still fails in openbsd. Default timeout in `CI` environments is 10.5 seconds. This has to be a high number in local environments.